drivers/serial/uart_ns16550.c

```diff
@@ -249,7 +264,7 @@ struct uart_ns16550_device_config {
 #if defined(CONFIG_PINCTRL)
 	const struct pinctrl_dev_config *pincfg;
 #endif
-#if defined(CONFIG_UART_NS16550_ACCESS_IOPORT) || defined(CONFIG_UART_NS16550_SIMULT_ACCESS)
+#if UART_NS16550_IOPORT_ENABLED
```

Comment:
please check some platform still using CONFIG_UART_NS16550_ACCESS_IOPORT ?

Reply:
> please check some platform still using CONFIG_UART_NS16550_ACCESS_IOPORT ?

I did a quick grep in the tree and couldn't find any remaining users (of course it's possible there are out-of-tree platforms, but we can't do much about those)